src/content/decentralized-social-networks.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Decentralized social media networks
+description: An overview of decentralized social networks on Ethereum
+lang: en
+template: use-cases
+emoji: ":mega:"
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Blockchain-based platforms for social interaction and content creation and distribution.  
+summaryPoint2: Decentralized social media networks protect user privacy and enhance data security. 
+summaryPoint3: Ethereum's ERC-20 tokens and NFTs are creating new avenues for content monetization. 
+---
+Today, social media platforms play a huge role in how we communicate and interact with each other. Social networking services have gone beyond mere fads and now underpin the digital economy. 
+
+However, centralized control of these platforms has created many problems. Data breaches, server outages, deplatforming, censorship, privacy violations—these are some of the trade-offs social media users are forced to accept. 
+
+To combat these issues, developers are building novel social media applications on decentralized networks like Ethereum. Decentralized social networks are poised to fix the the ills of legacy social platforms and improve the overall user experience. 
+
+## What are decentralized social networks? {#what-are-decentralized-social-networks}
+Decentralized social media networks are blockchain-based platforms that allow users to exchange information as well as publish and distribute content to audiences. Because these applications run on the blockchain, they are decentralized and resistant to censorship and undue control. 
+
+Many decentralized social networks exist as alternatives to established social media services, such as Facebook, LinkedIn, Twitter, and Medium. But blockchain-powered social networks have a number of features that put them ahead of traditional social platforms. 
+
+## Why do decentralized social media platforms matter? {#decentralized-social-media-benefits}
+
+To understand the benefits of decentralized social media networks, it's important to compare them against traditional social networks:
+
+| Decentralized social media                                                                                                      | Traditional social media                                                                 |
+| ----------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
+| Information is stored on the blockchain.                                                                                | Information is stored in servers and data centers.                                              |
+| Global distribution of nodes reduces the possibility of outages.                                                        | Highly susceptible to network failure and service downtimes.                                    |
```

Comment:
What I had in mind here was to compare the underlying infrastructure and how it affects operations. Specifically, I was thinking of the Facebook/Instagram/WhatsApp outage that happened last year. 

Of course, these outages don't happen very often, but the possibility is higher, since they have less distributed servers. It'll take something like the world's Internet going down or some black swan event to crash Ethereum, so by extension, platforms built on it can benefit from fault-tolerance. 

Thought I'll share my thought process around the point.

Reply:
Your reasoning makes sense to me but can we drop the hyperbole 😬?

Reply:
Yes, that makes sense. Like I said in another comment, I'll simply focus on highlighting the pros (w/o hyperbole) of decentralized social media without directly pitting them against traditional SM platforms.